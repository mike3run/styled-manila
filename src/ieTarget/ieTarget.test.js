import React from 'react'
import styled from 'styled-components'
import renderer from 'react-test-renderer'
import 'jest-styled-components'
import { ieTarget } from '..'

test('ieTarget works!', () => {
  const Test = styled.div`
    ${ieTarget`
      text-decoration: underline;
    `}
  `
  const tree = renderer.create(<Test />).toJSON()
  expect(tree).toMatchSnapshot()
  expect(tree).toHaveStyleRule('text-decoration', 'underline', {
    media: 'all and (-ms-high-contrast:none),(-ms-high-contrast:active)'
  })
})
