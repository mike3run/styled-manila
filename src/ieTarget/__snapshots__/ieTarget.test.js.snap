// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ieTarget works! 1`] = `
@media all and (-ms-high-contrast:none),(-ms-high-contrast:active) {
  .c0 {
    -webkit-text-decoration: underline;
    text-decoration: underline;
  }
}

<div
  className="c0"
/>
`;
